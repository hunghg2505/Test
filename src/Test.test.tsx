import TestRenderer from 'react-test-renderer';
import Test from './Test';

it('Test App', () => {
  const component = TestRenderer.create(<Test />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
