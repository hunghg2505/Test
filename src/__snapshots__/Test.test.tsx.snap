// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test App 1`] = `
<h1>
  Hello Test
</h1>
`;
